userspace/libscap/scap.h

```diff
@@ -324,11 +324,21 @@ typedef struct _scap_machine_info
 	uint64_t max_pid; ///< Highest PID number on this machine
 	char hostname[128]; ///< The machine hostname
 	uint64_t boot_ts_epoch; ///< Host boot ts in nanoseconds (epoch)
-	uint64_t reserved2; ///< reserved for future use
+	uint64_t flags; ///< flags
 	uint64_t reserved3; ///< reserved for future use
-	uint64_t reserved4; ///< reserved for future use
+	uint64_t reserved4; ///< reserved for future use, note: because of scap file captures needs to remain uint64_t, use flags if possible
 }scap_machine_info;
 
+/*!
+  \brief Agent information, not intended for scap file use
+*/
+typedef struct _scap_agent_info
+{
+	uint64_t start_ts_epoch; ///< Agent start timestamp, stat /proc/self/cmdline approach, unit: epoch in nanoseconds
+	double start_time; ///< /proc/self/stat start_time divided by HZ, unit: seconds
```

Comment:
Since the unit is seconds, I think we could make this `uint64_t` too and round-up the division.

Reply:
@jasondellaluce a bit worried about precision loss in general whenever we do a division which up until now we are not doing much in the project. For example see https://gitlab.com/procps-ng/procps/-/blob/master/library/pids.c#L286. Would it be ok to keep it as double as I would want to keep elapsed time of the process as double as well, because that will be used for division in CPU usage calculation at a later point in time.